utils/parse.js

```diff
@@ -55,16 +84,25 @@ exports.default = function parse(path, content, context) {
       console.warn(
         '`parseForESLint` from parser `' +
           parserPath +
-          '` is invalid and will just be ignored'
+          '` is invalid and will just be ignored',
+        path
```

Comment:
```suggestion
          '` is invalid and will just be ignored'
```
if the path is needed, let's put it in the string